.github/workflows/security-scans.yml

```diff
@@ -116,12 +108,52 @@ jobs:
           fetch-depth: 0
           submodules: 'true'
 
-      - name: Download manifest files
+      - name: Download tag
         id: download_manifests
         uses: actions/download-artifact@v3
         with:
-          name: manifests
-          path: scans
+          name: tag
+          path: tag
+
+      - name: Setup helm charts
+        run: |
+          helm repo add keptn "https://charts.lifecycle.keptn.sh"
+          helm repo update
+
+          for chart_dir in ./lifecycle-operator/chart \
+                  ./metrics-operator/chart \
+                  ./klt-cert-manager/chart \
+                  ./chart; do
+              cd "$chart_dir"
+              echo "updating charts for" $chart_dir
+              helm dependency update
+              helm dependency build
+              cd -  # Return to the previous directory
+          done
+
+      - name: Generate manifests
+        run: |
+          # Fetch tag of the images
+          TAG=$(ls tag/)
+          echo "$TAG"
+          mkdir scans
+
+          helm template keptn-test --namespace helmtests -f ./chart/values.yaml ./chart \
```

Comment:
I am working on a common value file to use in the install actions, maybe it would be worth to wait for that one instead of copypasting the pipeline here 🤔

Reply:
ok, we can do that, is it expected to be finished in the next few days or are we speaking about a longer time ?

Reply:
https://github.com/keptn/lifecycle-toolkit/pull/2361